bodhi/server/util.py

```diff
@@ -16,47 +16,50 @@
 Random functions that don't fit elsewhere
 """
 
+from collections import defaultdict
+from contextlib import contextmanager
+from datetime import datetime
+from os.path import join, dirname, basename, isfile
+import collections
+import functools
+import hashlib
 import os
-import arrow
+import pkg_resources
 import socket
-import urllib
-import tempfile
-import markdown
-import requests
 import subprocess
-import libravatar
-import hashlib
-import collections
-import pkg_resources
-import functools
-import transaction
-
-from os.path import join, dirname, basename, isfile
-from datetime import datetime
-from collections import defaultdict
-from contextlib import contextmanager
+import tempfile
+import urllib
 
-from sqlalchemy import create_engine
-from sqlalchemy.orm import scoped_session, sessionmaker
-from zope.sqlalchemy import ZopeTransactionExtension
 from pyramid.i18n import TranslationStringFactory
 from pyramid.settings import asbool
+from sqlalchemy.orm import scoped_session, sessionmaker
+from zope.sqlalchemy import ZopeTransactionExtension
+import arrow
+import libravatar
+import markdown
+import requests
+import transaction
 
-from . import log, buildsys
-from .exceptions import RepodataException
-from .config import config
+from bodhi.server import log, buildsys
+from bodhi.server.config import config
+from bodhi.server.exceptions import RepodataException
 
 try:
     import rpm
 except ImportError:
     log.warning("Could not import 'rpm'")
 
+
 _ = TranslationStringFactory('bodhi')
 
-## Display a given message as a heading
-header = lambda x: u"%s\n     %s\n%s\n" % ('=' * 80, x, '=' * 80)
 
-pluralize = lambda val, name: val == 1 and name or "%ss" % name
+def header(x):
+    """Display a given message as a heading."""
+    return u"%s\n     %s\n%s\n" % ('=' * 80, x, '=' * 80)
+
+
+def pluralize(val, name):
+    return val == 1 and name or "%ss" % name
```

Comment:
that got me confused a little, still kinda do, but somehow seems to work

(misses the docstring btw)